api/src/server.ts

```diff
@@ -62,6 +56,7 @@ const start = async () => {
 
   io.on('connection', socket => {
     socket.on('meeting:join', async meetingId => {
+      // @ts-ignore
```

Comment:
Should there be more explanation in the code about why we're using @ts-ignore for this section?

Reply:
Added documentation.